cfgov/v1/models/browse_filterable_page.py

```diff
@@ -12,7 +12,6 @@
 
 from .base import CFGOVPage
 from .learn_page import AbstractFilterPage
-from .. import forms
```

Comment:
This had to be removed and moved down into various `get_form_class` methods because otherwise there would be an infinite import loop involving `v1.models`, `v1.forms`, and `v1.util`. I've added a few additional unit tests to validate this change (see below).